src/ts/components/seekbar.ts

```diff
@@ -689,10 +693,6 @@ export class SeekBar extends Component<SeekBarConfig> {
       e.preventDefault();
 
       if (seeking) {
-        // During a seek (when mouse is down or touch move active), we need to stop propagation to avoid
-        // the VR viewport reacting to the moves.
-        e.stopPropagation();
-        // Because the stopped propagation inhibits the event on the document, we need to call it from here
```

Comment:
This comment indicates that the next call was only present because of the propagation stop. Are you sure it makes sense to keep it?

Reply:
The method `mouseTouchMoveHandler` has the `e.stopPropagation();` call. Doesn't make sense to have the method being called 2 times. Same can be discussed regarding `e.preventDefault()`